R/data.r

```diff
@@ -62,3 +62,22 @@
 #'   \item{model}{the name of the model}
 #' }
 "stat_forecasts"
+
+#' Flusight ILI data for the USA starting with the 2003/2004 season through the
```

Comment:
```suggestion
#' Flusight ILI data for the US starting with the 2003/2004 season through the
```

I generally refer to is as US, which is maybe not fully correct, but just to be consistent here, am making this suggestion.